test/unit/context/context_mm_black.h

```diff
@@ -40,7 +40,9 @@ class ContextBlack : public CxxTest::TestSuite {
   }
 
   void testPushPop() {
-#ifdef CVC4_CONTEXT_MEMORY_MANAGER
+#ifdef CVC4_DEBUG_CONTEXT_MEMORY_MANAGER
+#warning "Using the debug context memory manager, omitting unit tests"
+#else /* CVC4_DEBUG_CONTEXT_MEMORY_MANAGER */
```

Comment:
I find the comment here a little confusing, it kinda implies that the code below is enabled when `CVC4_DEBUG_CONTEXT_MEMORY_MANAGER` is defined.